sklego/pipeline.py

```diff
@@ -0,0 +1,151 @@
+'''
```

Comment:
I think this docstring is added on a module level and since it is possible that we might have other pipelines in the future (like the `BoosterPipeline`) it might be best to describe the module in a more general setting.

Reply:
Done.